omod/src/main/resources/liquibase.xml

```diff
@@ -195,5 +195,31 @@
 							 tableName="metadatamapping_metadata_source"
 							 columnNames="name" />
 	</changeSet>
-
+	
+	<changeSet id="metadatamapping-2016-01-06-0800" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="metadatamapping_metadata_term_mapping"
+						  columnName="metadata_class" />
+		</preConditions>
+		<comment>
+			Make MetadataTerm.metadataClass optional
```

Comment:
If you want to refer to the class name here, it is actually MetadataTermMapping.